src/components/daydream-controls.js

```diff
@@ -83,7 +83,9 @@ module.exports.Component = registerComponent('daydream-controls', {
   },
 
   checkIfControllerPresent: function () {
-    var isPresent = this.isControllerPresent(this.el.sceneEl, GAMEPAD_ID_PREFIX, {hand: this.data.hand});
+    var isPresent = this.hands.reduce(function (present, hand) {
+      return present && this.isControllerPresent(this.el.sceneEl, GAMEPAD_ID_PREFIX, {hand: hand});
+    }.bind(this));
```

Comment:
a little different from prior discussion; this will fail to indicate presence if gamepad doesn't include hand property, rather than not filtering if default value is provided... which is presumably why you needed to go into tracked-controls?